scripts/generate-protocol.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+set -e
+
+if ! type -P protoc >/dev/null 2>&1; then
+  printf '%s\n' 'protoc is not available' 1>&2
```

Comment:
I think it is much simpler to use the dockerized version. So people don't need to try to get some "random" binary on the system. This will also allow us to pin the tool version.

Reply:
I didn't know about this existing image, where is it?
Or are you talking about _creating_ a Dockerfile to generate an image for it?

Reply:
Oh, looks like I confused it with something :thinking:. I can't find an official image for it. So, this script is fine.
Thanks for it btw ;)

Reply:
uch! I already removed it :)
but I think we could actually add a small `Dockerfile` for simplifying the _generation_ of the compiled Haskell files from protobuf definitions.

Reply:
I reintroduced, for now, the `generate-protocol.sh` script.